api/config.py

```diff
@@ -52,27 +52,28 @@
     },
 }
 
-__config = copy.deepcopy(DEFAULT_CONFIG)
+def apply_env_variables(config):
+    # Overwrite default config values with SCITRAN env variables if available
+    for outer_key, scoped_config in config.iteritems():
+        if outer_key in ['core', 'site', 'auth', 'persistent']:
```

Comment:
What's the purpose of this check? I wouldn't anticipate a code change when adding a new outer config key.